docs/hopr-documentation/docs/developers/tutorial-hello-world.md

```diff
@@ -174,10 +174,12 @@ information to send a message from `node 1`. Make sure to keep your WebSocket cl
 To send a message from `node 1` to `node 2`, we need to use `node 1`'s REST API, specifically the `/messages` endpoint. Using `curl`
 or any other HTTP client, send the following request:
 
+Replace `$HOPR_NODE_2_ADDR` with `node 2` address.
+
 ```bash
 echo -n $apiToken | base64 | xargs -I {} curl -s -H "Authorization: Basic {}" \
 -H 'Content-Type: application/json' \
--d '{"body":"Hello world","recipient":"16Uiu2HAmKhrwGWcvaZ3ic5dgy7oFawmnELJGBrySSsNo4bzGBxHW"}' \
+-d '{"body":"Hello world","recipient":"$HOPR_NODE_2_ADDR"}' \
 $HOPR_NODE_1_HTTP_URL/api/v2/messages
```

Comment:
Actually, this can be simplified to:
```bash
curl -X 'POST' \
  "$HOPR_NODE_1_HTTP_URL/api/v2/messages/" \
  -H 'accept: */*' \
  -H "x-auth-token: $apiToken" \
  -H 'Content-Type: application/json' \
  -d "{
  \"body\": \"Hello world from Node 1\",
  \"recipient\": \"$HOPR_NODE_2_ADDR\"
}"
```